partiql-ast/src/main/kotlin/org/partiql/ast/domains/AsAnyElement.kt

```diff
@@ -0,0 +1,13 @@
+@file:JvmName("Utils")
+
+package org.partiql.ast.domains
+
+import com.amazon.ionelement.api.IonElement
+
+// FIXME: Workaround for a bug in PIG that is fixed in its next release:
+//  https://github.com/partiql/partiql-ir-generator/issues/41
+// FIXME: This does NOT conform to PartiQL's standards of forbidding top-level functions/variables. However,
+//  given the issue above, this is required. To reduce some of the grossness of top-level functions, we have specified
+//  the file's JvmName further above.
+internal fun List<IonElement>.asAnyElement() =
+    this.map { it.asAnyElement() }
```

Comment:
I can do a PIG release today so we don't have to keep this around.

Reply:
Works for me! LMK when you do and I'll remove.

Reply:
And PIG plugin release since we are actually a consumer of the plugin .. which is a consumer of PIG.

Reply:
I believe we can make these changes now because moving these domains without repackaging is not a breaking change.

Reply:
Alright, with the plugin release, I've now set the target to `main` and have removed the utility function.